### DISCUSSION TITLE
In the GPTMD mode, what is the difference between phosphoserine and phosphorylation on S? The same is for threonine and tyrosine

### DISCUSSION THREAD

**Discussion by AmirWIS** [smith-chem-wisc/MetaMorpheus#2356] [Q&A] [answered]
In the GPTMD mode, what is the difference between phosphoserine and phosphorylation on S? The same is for threonine and tyrosine

**Comment by nbollis** [comment, marked as answer]
There is no functional difference between the two. The main point here is that phosphoserine comes directly from unitprot annotations whereas phosphorylation on S was a modification added by MetaMorpheus. The difference is in notation, not chemistry.
